tests/test_request_left.py

```diff
@@ -0,0 +1,60 @@
+from twisted.internet import defer
+from twisted.trial.unittest import TestCase
+from scrapy.signals import request_left_downloader
+from scrapy.spiders import Spider
+from scrapy.utils.test import get_crawler
+from tests.mockserver import MockServer
+
+
+class SignalCatcherSpider(Spider):
+    name = 'signal_catcher'
+
+    def __init__(self, crawler, url, *args, **kwargs):
+        super(SignalCatcherSpider, self).__init__(*args, **kwargs)
+        crawler.signals.connect(self.on_response_download,
+                                signal=request_left_downloader)
+        self.catched_times = 0
```

Comment:
```suggestion
        self.caught_times = 0
```
The subsequent occurrences should be updated too.